### ISSUE TITLE
Not compatible with ember-cli 0.1.1

### ISSUE THREAD

**Issue by Globegitter** [kurko/ember-sync#21]
If you install the npm package with the latest ember-cli it throws an error that loader.js is a file as well as a folder or something.
Installing via bower and setting things up manually works.

**Comment by csprocket777** [comment]
Any updates on this? I'm running into this now and don't know how to solve it.

**Comment by kurko** [comment]
@csprocket777 can you provide more details like versions etc? I'm running on 0.1.5 without problems.

**Comment by csprocket777** [comment]
I just went through uninstalled ember-cli, node, npm, bower, etc. Did a fresh install of everything, upgraded my project to ember-cli 0.1.7.

Same issue.

```
version: 0.1.7
Could not find watchman, falling back to NodeWatcher for file system events
Livereload server on port 35729
Serving on http://0.0.0.0:4200/
Merge error: conflicting file types: loader.js is a directory in /Users/chuckshipman/Documents/TM_git_repos/recruiter/tmp/custom_static_compiler-tmp_dest_dir-9zj7Ez2L.tmp but a file in /Users/chuckshipman/Documents/TM_git_repos/recruiter/tmp/tree_merger-tmp_dest_dir-Lh8dLEnd.tmp
Remove or rename either of those.
Error: Merge error: conflicting file types: loader.js is a directory in /Users/chuckshipman/Documents/TM_git_repos/recruiter/tmp/custom_static_compiler-tmp_dest_dir-9zj7Ez2L.tmp but a file in /Users/chuckshipman/Documents/TM_git_repos/recruiter/tmp/tree_merger-tmp_dest_dir-Lh8dLEnd.tmp
Remove or rename either of those.
    at mergeRelativePath (/Users/chuckshipman/Documents/TM_git_repos/recruiter/node_modules/ember-cli/node_modules/broccoli-merge-trees/index.js:87:21)
    at /Users/chuckshipman/Documents/TM_git_repos/recruiter/node_modules/ember-cli/node_modules/broccoli-merge-trees/index.js:23:5
    at $$$internal$$tryCatch (/Users/chuckshipman/Documents/TM_git_repos/recruiter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:490:16)
    at $$$internal$$invokeCallback (/Users/chuckshipman/Documents/TM_git_repos/recruiter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:502:17)
    at $$$internal$$publish (/Users/chuckshipman/Documents/TM_git_repos/recruiter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:473:11)
    at Object.$$rsvp$asap$$flush [as _onImmediate] (/Users/chuckshipman/Documents/TM_git_repos/recruiter/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1581:9)
    at processImmediate [as _immediateCallback] (timers.js:354:15)
```

running "ember -v" gives me this:
```
version: 0.1.7
Could not find watchman, falling back to NodeWatcher for file system events
node: 0.10.35
npm: 2.1.8
```

**Comment by csprocket777** [comment]
I also just tried downgrading ember-cli to 0.1.5 but still no luck.
Is there a specific version of the loader I'm supposed to be using?

**Comment by kurko** [comment]
@csprocket777 that things is that `loader.js` was renamed to `loader`. I'm updating my app soon to the latest EmberCLI and will certainly face this problem. Meanwhile, you can do one of the following:

1. rename `loader.js` directory to `loader` locally
2. send a PR to fix that
3. wait until I get there myself and open a PR.

If you could do (2), I'd be forever grateful :grin:

**Comment by csprocket777** [comment]
See PR: https://github.com/kurko/ember-sync/pull/27

**Comment by jakecraige** [comment]
fixed from #25
